src/Engine/Components/Trace/EngineTracePlayer.cpp

```diff
@@ -82,7 +82,7 @@ void EngineTracePlayer::checkAfterLoadRng(int expectedRandomState) {
     if (_flags & TRACE_PLAYBACK_SKIP_RANDOM_CHECKS)
         return;
 
-    int randomState = grng->peek(1024);
+    int randomState = grng->peek(1024 * 1024);
```

Comment:
Some global constant is not needed?

Reply:
I'll drop this one completely in the next few PRs.